articles/policies/rate-limits.md

```diff
@@ -196,7 +196,13 @@ The following Auth0 Management API endpoints return rate limit-related headers.
 
 ### Authentication API
 
-The following Auth0 Authentication API endpoints return rate limit-related headers.
+The rate limits for this API differ depending on whether your tenant is free or paid.
+
+::: note
+For all __free tenants__, Authentication API usage is restricted to 300 requests per minute. Note that the limit is set by tenant and not by endpoint.
```

Comment:
Would we link to the "long" page from here?

Reply:
If we decide to use both (my particular preference) then yes.